app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+    def can_be_cleaned(self, mark: int) -> bool:
+        return mark > self.clean_mark
+
+    def update_mark(self, mark: int) -> None:
+        if self.can_be_cleaned:
```

Comment:
You should call the can_be_cleaned function to get the result
```suggestion
        if self.can_be_cleaned(mark):
```